js/workers/pjs/jshint-worker.js

```diff
@@ -6,15 +10,6 @@ self.onmessage = function(event) {
     // we just use the normal file.
     if (!init) {
         init = true;
-
-        importScripts(event.data.externalsDir +
-            "es5-shim/es5-shim.js?cachebust=" + (new Date()).toDateString());
-
-        importScripts(event.data.jshintFile +
-            "?cachebust=" + (new Date()).toDateString());
-
-        importScripts(event.data.externalsDir +
-            "underscore/underscore.js?cachebust=" + (new Date()).toDateString());
```

Comment:
Why can we get rid of these `importScripts`?

Reply:
We now load in the built worker file, so that entry point includes es5-shim and JSHint.
JSHint never used underscore. It looks like your debugger PR added underscore, and then we didn't later remove it:
https://github.com/Khan/live-editor/commit/10f0c17a3c410b9b995b9253097bf8ab8e6821a6